integrations/gsheets/src/definitions/index.ts

```diff
@@ -0,0 +1,30 @@
+import type { IntegrationDefinitionProps } from '@botpress/sdk'
+import z from 'zod'
+
+import { actions } from './actions'
+import { channels } from './channels'
+
+export { actions }
+export { channels }
+
+export const configuration = {
+  schema: z.object({
+    spreadsheetId: z
+      .string()
+      .describe('The ID of the Google Spreadsheet to interact with'),
+    privateKey: z
```

Comment:
**question:** how do I get this ? I tried using the api key provided here:

![image](https://github.com/botpress/botpress/assets/25970722/29585242-92ea-4cb5-b072-4a0d826cf4d4)

But then I got: `'Get Values' exception Error: error:1E08010C:DECODER routines::unsupported`